app/src/main/java/nodomain/freeyourgadget/gadgetbridge/service/DeviceCommunicationService.java

```diff
@@ -784,6 +785,9 @@ public void onSharedPreferenceChanged(SharedPreferences sharedPreferences, Strin
                 mDeviceSupport.setAutoReconnect(autoReconnect);
             }
         }
+        if (key.equals("chart_max_heart_rate") || key.equals("chart_min_heart_rate")) {
```

Comment:
Use the constant from `GBPrefs` here? :-)

Reply:
yes you are correct <.< i shouldn't miss such things.
thanks